src/stacked_borrows.rs

```diff
@@ -380,58 +426,66 @@ impl<'tcx> Stack {
     fn access(
         &mut self,
         access: AccessKind,
-        tag: SbTag,
+        tag: Option<SbTag>,
         (alloc_id, alloc_range, offset): (AllocId, AllocRange, Size), // just for debug printing and error messages
         global: &mut GlobalStateInner,
         current_span: &mut CurrentSpan<'_, '_, 'tcx>,
         alloc_history: &mut AllocHistory,
+        exposed_tags: &FxHashSet<SbTag>,
     ) -> InterpResult<'tcx> {
         // Two main steps: Find granting item, remove incompatible items above.
```

Comment:
In fact, maybe this is the right place to separately handle `tag == None`? I am not sure if there is much code that can be shared between those cases. We might want a separate `access_wildcard` function and put all the wildcard logic in there.